.cirrus.yml

```diff
@@ -238,7 +238,7 @@ task:
       <<: *FIREBASE_SHARD_TEMPLATE
 
     - name: web_smoke_test
-      only_if: "changesInclude('.cirrus.yml', 'examples/hello_world/**' ,'dev/**', 'packages/flutter/**', 'packages/flutter_test/**', 'packages/flutter_tools/lib/src/test/**', 'packages/flutter_web_plugins/**', 'bin/**') || $CIRRUS_PR == ''"
+      only_if: "changesInclude('.cirrus.yml', 'examples/hello_world/**' ,'dev/**', 'packages/flutter/**', 'packages/flutter_test/**', 'packages/flutter_tools/lib/src/test/**', 'packages/flutter_web_plugins/**', 'bin/**') || $CIRRUS_PR != ''"
```

Comment:
these diffs are always so hard for me to read...

So this is saying we always want to run the test for any PR, and on post-submit only on affected diffs...right?

This doesn't seem like the right logic. Should it be `&& $CIRRUS_PR == ''`, which would be only run pre-submit with a diff on the affected files?

Reply:
I see, yeah I think it has to be && but I thought $CIRRUS_PR == '' meant postsubmit.

Reply:
lol, yeah you're right. so `&& $CIRRUS_PR != ''`

Reply:
This is why I put question marks at the end of all my statements, I never get this right the first try.

Reply:
yeah, errors in those conditionals are usually difficult to spot. Updated.